examples/Workflows/nmt_tranformers_pipeline/README.md

```diff
@@ -0,0 +1,123 @@
+# Workflow pipeline example using nmt transformer nlp model
+
+This example uses the existing [nmt_transformers](../../nmt_transformer) standalone example to create a workflow. We use three models, in two examples to demonstrate stringing them together in a workflow.
+To change the default batch size and batch delay the yaml file for the workflow can to be changed. This cannot currently be set via the REST API.
+
```

Comment:
@maaquib please add some explanation about each example (e.g, dual translation, back translation) and similar to comments on CV example some clarification on the flow in the DAG/ aggregating function.